### ISSUE TITLE
[MAS1.4.3] [Visual Requirements -Settings] Color contrast ratio for “User ID” element is less than 4.5:1.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1964]
**Actual Result:**
Color contrast ratio of “User ID” element is less than 4.5:1.

**Expected Result:**
The color contrast ratio should be greater than or equal to 4.5:1 for “User ID” element.

 **Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to any already created bot and select any bot..
4. "Live Chat" screen will appear.
5. Navigate to "Edit ngrok settings" button and activate it.
6. Emulator setting screen gets opened.
7. Now, navigate to "User ID" element.
8. Open 'Color Contrast Analyzer' application.
9. Verify the Color Contrast Ratio is less than 4.5:1 for “User ID” element.

**Note:**
Same issue found with the time information in the live chat window. Please refer the attachment: "CCA failed for time info" for the reference. 
Repro Path:- Live Chat-->Message Time
Same issue got fixed for Windows. Refer Bug: [https://github.com/microsoft/BotFramework-Emulator/issues/1766](url)

**User Impact:**
Users with low vision or color blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)

**Attachment:**
![CCA failed for User ID element](https://user-images.githubusercontent.com/55279608/68108285-670be000-ff0d-11e9-97f2-31c4b6ea4300.png)
![CCA failed for time info](https://user-images.githubusercontent.com/55279608/68385068-fff56200-017e-11ea-8c20-f12a724c53de.png)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
